### DISCUSSION TITLE
Windows VHD Plugin: Encrypting VHD with Veracrypt

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1370] [Q&A]
Is it possible to have the Windows VHD contents encrypted with Veracrypt disk encryption? I tried to encrypt the drive and I tried selection the Windows partition only option and the whole drive option and for both options when it does the reboot test to ensure Veracrypt disk encryption will work before encryption the system the test fails . I was not prompted a decrypt unlock screen when I rebooted into the Ventoy Windows VHD.

Is there a way to install and encrypt the Windows VHD on Ventoy using Veracrypt?

**Comment by trymeouteh** [comment]
I tried to install and encrypt the system drive inside a Virtualbox VM and it test did work. It must have something to do with Ventoy boot loader that causes this to not work.

**Comment by bingoxo** [comment]
any update to this ?

i'm reluctant to install veracrypt on my vhd fearing it  won't work
